### ISSUE TITLE
Suggestions for users in the U.S.

### ISSUE THREAD

**Issue by amarder** [butor/blackbird#313]
It looks like Bitfinex no longer supports U.S. individual users: https://www.bitfinex.com/posts/227

Would you suggest trying to use Poloniex or GDAX in its place?

**Comment by amarder** [comment]
It looks like Poloniex does extremely badly with the Better Business Bureau: https://www.bbb.org/delaware/business-reviews/foreign-currency-exchange-broker/poloniex-inc-in-wilmington-de-92004252

**Comment by greatwolf** [comment]
It depends, are you only trading between other cryptos or do you need to trade between bitcoin and fiat? Note that bitfinex, polo and bittrex are trading USDT tethers and not actual dollars. There's some rumors going around that Tether may not have real US dollars to fully back the tethers created.

For fiat, GDAX and Gemini or probably the main choices.

**Comment by amarder** [comment]
Thanks!

My (weak) understanding of how blackbird is set up, is that I want to buy bitcoin on one exchange and short sell it on another. Since U.S. individuals are no longer supported on Bitfinex I was looking at the other exchanges that support short selling.

Unfortunately, to short sell on GDAX it looks like I need to have at least five million dollars in my account! https://support.gdax.com/customer/portal/articles/2725812-what-is-an-eligible-contract-participant-ecp-

**Comment by greatwolf** [comment]
Yeah, it's an unfortunate reality. We have government regulation to thank for that.

**Comment by johnjjung** [comment]
https://www.kraken.com/en-us/help/api#private-user-trading

It seems like with the exit if bitfinex there’s no other options for short positions unless you’re an institutional investor on gdax (which I’m sure most of us here are not).

Has anyone tried implementing the kraken short positions? Then it would allow for US exchange accounts to work with this. I’m gonna give it a shot but I just want to make sure I’m not doing redundant work and if anyone can help me with tests that’d be great

**Comment by itamarro** [comment]
What about implementing it via positions in derivatives, e.g. in Deribit? Has anyone taken a look at this possibility?

**Comment by itamarro** [comment]
@johnjjung one note is that right now Kraken is not reliable for any sort of automated trading, hence why I would not personally put any work in supporting it. They are working on it and perhaps this would change soon, but as it stands they are openly admitting the situation is untenable: https://blog.kraken.com/post/1399/degraded-service-upgrade-next-week/, https://blog.kraken.com/post/1413/update-on-system-performance-and-upgrade/

**Comment by johnjjung** [comment]
@itamarro hmm, thanks, I looked into this and agree that their API isn't stable. I started to try to fetch ticker data but it 502's out pretty often. Will pick this back up later
